### PR TITLE
Add Huobi Spot exchange

### PR DESCRIPTION
**Pull request by artememelin**
### Pull request checklist

- [x] Did you bump the project version?
- [x] Did you add a description to the Pull Request?
- [ ] Did you add new tests?
- [x] Did you add reviewers?

### CODE REVIEW

File: src/Huobi/Huobi.jl [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+    return ""
+end
+
+function CryptoAPIs.request_query(query::Q)::String where {Q<:HuobiCommonQuery}
+    return Serde.to_query(query)
+end
+
+function CryptoAPIs.request_headers(client::HuobiClient, ::HuobiCommonQuery)::Vector{Pair{String,String}}
+    return Pair{String,String}["Content-Type"=>"application/json"]
+end
```
Comment (dmitrii-doronin) [inline src/Huobi/Huobi.jl:158]:
```suggestion
function CryptoAPIs.request_headers(client::HuobiClient, ::HuobiCommonQuery)::Vector{Pair{String,String}}
    return Pair{String,String}[
        "Content-Type"=>"application/json",
    ]
end
```
```suggestion
function CryptoAPIs.request_headers(client::HuobiClient, ::HuobiCommonQuery)::Vector{Pair{String,String}}
    return Pair{String,String}["Content-Type"=>"application/json"]
end
```

File: src/Huobi/Spot/API/Candle.jl [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+using CryptoAPIs.Huobi
+using CryptoAPIs.Huobi: Data
+using CryptoAPIs: Maybe, APIsRequest
+
+@enum TimeInterval m1 m5 m15 m30 m60 h4 d1 M1 Y1
+
+Base.@kwdef struct CandleQuery <: HuobiPublicQuery
+    period::TimeInterval
+    symbol::String
+    size::Int64 = 150       # The number of data returns default 150 [1-2000]
```
Comment (dmitrii-doronin) [inline src/Huobi/Spot/API/Candle.jl:19]:
```suggestion
    size::Int64 = 150       #  Default size 150 [1-2000]
```
```suggestion
    size::Int64 = 150       # The number of data returns default 150 [1-2000]
```

File: src/Huobi/Spot/API/Candle.jl [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    x == M1  && return "1mon"
+    x == Y1  && return "1year"
+end
+
+struct CandleData <: HuobiData
+    amount::Maybe{Float64}
+    close::Maybe{Float64}
+    count::Maybe{Int64}
+    high::Maybe{Float64}
+    id::Maybe{NanoDate}
```
Comment (dmitrii-doronin) [inline src/Huobi/Spot/API/Candle.jl:39]: Id seems to be unique

File: src/Huobi/Spot/API/CommonSymbol.jl [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+
+function Serde.deser(::Type{CommonSymbolData}, ::Type{<:Maybe{Time}}, x::AbstractString)::Time
+    return Time(x)
+end
+
+function Serde.deser(::Type{CommonSymbolData}, ::Type{<:Maybe{SymbolPartition}}, x::String)::SymbolPartition
+    x == "main"       && return main
+    x == "innovation" && return innovation
+    x == "st"         && return st
+end
```
Comment (dmitrii-doronin) [inline src/Huobi/Spot/API/CommonSymbol.jl:76]: Not needed?

File: src/Huobi/Spot/API/CommonSymbol.jl [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+
+function Serde.deser(::Type{CommonSymbolData}, ::Type{<:SymbolState}, x::String)::SymbolState
+    x == "online"     && return online
+    x == "pre-online" && return pre_online
+    x == "offline"    && return offline
+    x == "suspend"    && return suspend
+end
```
Comment (dmitrii-doronin) [inline src/Huobi/Spot/API/CommonSymbol.jl:83]: Throw error IMO

File: src/Huobi/Huobi.jl [hunk 6]
```diff
@@ -0,0 +1,169 @@
...
+    return Serde.to_query(query)
+end
+
+function CryptoAPIs.request_query(::Q)::String where {Q<:HuobiPrivateQuery}
+    return ""
+end
+
+function CryptoAPIs.request_headers(client::HuobiClient, ::HuobiCommonQuery)::Vector{Pair{String,String}}
+    return Pair{String,String}[
+        "Content-Type"=>"application/json",
```
Comment (yshadrin) [inline src/Huobi/Huobi.jl:159]:
```suggestion
        "Content-Type" => "application/json",
```

**Pull request merged**